### PR TITLE
feat: AntD-based Card component (early/proto version)

### PR DESCRIPTION
**Pull request by rusackas**
### SUMMARY

This PR does a few things:
1) Adds to the webpack config, allowing unit tests to be run from alongside component files.
2) Adds a couple tweaks to React Storybook
3) The point of all this: a proto-component for Cards, using AntD. This will be expanded on to _actually_ use in the UI in subsequent PRs for CRUD list/card views.

image

### TEST PLAN

Tested/viewable in Storybook! `npm run storybook`
Basic unit tests build from storybook stories.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/.storybook/preview.jsx [hunk 1]
```diff
@@ -29,4 +31,14 @@ const themeDecorator = storyFn => (
...
+addDecorator(withPaddings);
+
+addParameters({
+  paddings: [
+    { name: 'None', value: '0px' },
+    { name: 'Small', value: '16px' },
+    { name: 'Medium', value: '32px', default: true },
+    { name: 'Large', value: '64px' },
+  ],
+});
```
Comment (mistercrunch) [inline superset-frontend/.storybook/preview.jsx:44]: I think the python pre-commit hooks get these EoF lints for me

**Comment by mistercrunch** [comment]
I'd like to see the prod bundle size (`npm run build`) before / after the antd import to make sure treeshaking is working right

**Comment by mistercrunch** [comment]
Overall the approach of wrapping all `antd` components is going to be a fair amount of boilerplate code, but I think it's worth it overall for the flexibility it offers:
- reducing the props to a subset of the ones exposed by antd helps simplifying the component interface surface, creates more visual consistency in Superset
- allows for rewiring/styling abstraction to be all in one place
- {more! just can't think of it just now}

**Comment by nruhe** [comment]
> reducing the props to a subset of the ones exposed by antd helps simplifying the component interface surface, creates more visual consistency in Superset

@mistercrunch Aside from the style props, we should generally avoid limiting the properties antd exposes to us. Doing so just makes the components more rigid and will lead to bad designs in the long run.

**Pull request closed without merging**